### ISSUE TITLE
Error in if (!out$check_pass)

### ISSUE THREAD

**Issue by joelnitta** [ropensci-review-tools/pkgcheck-action#34]
When [running the GH action](https://github.com/joelnitta/dwctaxon/actions/runs/4141768436/jobs/7161692566), I get this warning and error:

```
  Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
  Error in if (!out$check_pass) { : missing value where TRUE/FALSE needed
  Calls: checks_to_markdown ... FUN -> summarise_check -> do.call -> output_pkgchk_ci
  Execution halted
```

**Comment by mpadge** [comment]
@assignUser Any insights here? Made me realise that `actions/checkout` was still v2, so #35 might help. Wait to see what effect that might have first?

**Comment by assignUser** [comment]
The warning is unrelated to the issue as it does not cause any side effects for now (once they deprecate that, it will probably throw an error) but I will fix it in octolog.

The actual error happens within pkgcheck, I had a look at the action code maybe [here](https://github.com/ropensci-review-tools/pkgcheck-action/blob/main/R/check.R#L32) as that is the first time output is called after the group starts?

**Comment by mpadge** [comment]
I'm a bit stumped here. I can't replicate that error on any system, including the rOpenSci check runner. It seems to arise because of a missing value for `check_pass` in `output_pkgcheck_ci()`, but that value is defined here:
https://github.com/ropensci-review-tools/pkgcheck/blob/9b1cb266c22f42d81e1c549f6e7212a0c1665f6b/R/check-ci.R#L4
And will be set to `FALSE` even when fed no information at all, so I don't see how it can be undefined? If things work, it should then be redefined here:
https://github.com/ropensci-review-tools/pkgcheck/blob/9b1cb266c22f42d81e1c549f6e7212a0c1665f6b/R/check-ci.R#L10
but again, even if both `i` and `wf$conclusion` are somehow undefined, that will still define a value of `check_pass <- FALSE`.

The failure presumably arises a few lines later at:
https://github.com/ropensci-review-tools/pkgcheck/blob/9b1cb266c22f42d81e1c549f6e7212a0c1665f6b/R/check-ci.R#L19
... but we need a traceback to see how or where. @assignUser Do you have a local `act` set up you could use to step through the action?

**Comment by assignUser** [comment]
No I don't have act setup, the actions I usually work on don't really work with it. But the action runs entirely in a docker container so that should work fine locally with the repo mounted. I will have a look in a bit

**Comment by assignUser** [comment]
I was unable to reproduce the issue within the docker container...

**Comment by assignUser** [comment]
I tried again but was still unable to produce the error... which is not really what should happen using docker... :/

**Comment by mpadge** [comment]
@assignUser I think we can close this now, right?
